frontend/src/pages/Matching.vue

```diff
@@ -1,40 +1,480 @@
+
+          <template v-for="(column, index) in columns" v-if="index == 5">
+            <b-table-column :key="column.id" v-bind="column" sortable>
+              <template
+                v-if="column.searchable"
+                slot="searchable"
+                slot-scope="props"
+              >
+                <b-input
+                  v-model="props.filters[props.column.field]"
+                  icon="magnify"
+                  size="is-small"
+                />
+              </template>
+              <template v-slot="props">
+                {{ props.row.NativeLanguage  }}
+              </template>
+            </b-table-column>
+          </template>
+
+
+          <!-- English Proficiency -->
+
+          <template v-for="(column, index) in columns" v-if="index == 4">
+            <b-table-column :key="column.id" v-bind="column" sortable>
+              <template
+                v-if="column.searchable"
+                slot="searchable"
+                slot-scope="props"
+              >
+                <b-input
+                  v-model="props.filters[props.column.field]"
+                  icon="magnify"
+                  size="is-small"
+                />
+              </template>
+              <template v-slot="props">
+                {{ props.row.EnglishProficiency }}
+              </template>
+            </b-table-column>
+          </template>
+
+
+
+          <!-- Source -->
+
+          <template v-for="(column, index) in columns" v-if="index == 6">
+            <b-table-column :key="column.id" v-bind="column" sortable>
+              <template
+                v-if="column.searchable"
+                slot="searchable"
+                slot-scope="props"
+              >
+                <b-input
+                  v-model="props.filters[props.column.field]"
+                  icon="magnify"
+                  size="is-small"
+                />
+              </template>
+              <template v-slot="props">
+                {{ props.row.Source }}
+              </template>
+            </b-table-column>
+          </template>
+
+
+        </b-table>
+      </section>
+            <!--start of table 2 -->
+
+      <PageHeader slot="header" class="mid-white">
+          <div class="header-right">
+             <div class="subheader-left">Matches in progress</div>
+              <div class="subheader-right">
+                <button
+                  class="button field is-white"
+                  @click="checkedRows = []"
+                  :disabled="!checkedRows.length"
+                >
+                  <span>Unmatch Selected </span> ({{ checkedRows.length }})
+                </button>
+                <div class="divider"/>
+                <button
+                  class="button field is-white"
+                  @click="cardModal()"
+                  :disabled="!checkedRows.length"
+                >
+                  <span>Matched selected </span> ({{ checkedRows.length }})
```

Comment:
should be "confirm selected" as the button leads to a pop-up saying that matches will be confirmed + an email will be sent to teachers